packages/components/src/Form/Inputs/InputFilters/InputFiltersChip.tsx

```diff
@@ -33,14 +33,18 @@ interface InputFiltersChipProps
   onDelete: (field: FieldFilter) => void
 }
 
+/* Replace comma not followed by white space with comma followed by space */
+const defaultFormatValue = (value?: string) =>
+  value ? value.replace(/(,(?=\S)|:)/, ', ') : undefined
+
 export const InputFiltersChip: FC<InputFiltersChipProps> = forwardRef(
   ({ filter, onDelete, ...props }, ref: Ref<HTMLSpanElement>) => {
     const handleDelete = () => onDelete(filter)
     return (
       <Chip onDelete={handleDelete} prefix={filter.field} ref={ref} {...props}>
```

Comment:
Not related to this PR, but is there a reason the prefix wouldn't be the label?

```suggestion
      <Chip onDelete={handleDelete} prefix={filter.label} ref={ref} {...props}>
```

Reply:
That was per the original design – I suspect we might need to get even more specific and have a `filter.prefix` so that the specifier can be super-specific.